_posts/2015-09-18-epages-you-2015.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Annual company event: ePagees rock the YO(U)"
```

Comment:
Back in the days it was written (YO)U, because it started as ePages University. So not the "U" was optional, but the "YO". I don't know where the YO(U) version comes from.

In this day and age I would vote for dropping the parenthesis altogether.